src/cargo/util/style.rs

```diff
@@ -12,3 +12,20 @@ pub const GOOD: Style = AnsiColor::BrightGreen.on_default().effects(Effects::BOL
 pub const VALID: Style = AnsiColor::BrightCyan.on_default().effects(Effects::BOLD);
 pub const INVALID: Style = annotate_snippets::renderer::DEFAULT_WARNING_STYLE;
 pub const TRANSIENT: Style = annotate_snippets::renderer::DEFAULT_HELP_STYLE;
+
+pub const UPDATE_ADDED: Style = NOTE;
+pub const UPDATE_REMOVED: Style = ERROR;
+pub const UPDATE_UPGRADED: Style = GOOD;
+pub const UPDATE_DOWNGRADED: Style = WARN;
+pub const UPDATE_UNCHANGED: Style = anstyle::Style::new().bold();
+
+pub const DEP_NORMAL: Style = anstyle::Style::new().effects(anstyle::Effects::DIMMED);
```

Comment:
> My primary goal with this is having a central place to document this for other applications

Do you mean we'll have doc comments for each, or code itself is self-documented?

Ask because I don't think the variable names here describe the intent clear if not looking into the implementation

But anyway, having styles in one place is great.